src/utils.tsx

```diff
@@ -14,7 +14,7 @@ function useIdPolyfill() {
 
   useIsomorphicLayoutEffect(() => {
     if (id === undefined) {
-      setId(++ID)
+      setId(genId)
```

Comment:
this is good👍 we don't have to execute function !?

```suggestion
      setId(genId())
```

Reply:
@ChanhyukPark-Tech 
Thanks :)